cassandra-analytics-common/src/main/java/org/apache/cassandra/spark/utils/Properties.java

```diff
@@ -40,7 +40,9 @@ public final class Properties
     public static final int DEFAULT_MAX_RETRIES = 10;
     public static final long DEFAULT_MILLIS_TO_SLEEP = 500;
     public static final int DEFAULT_MAX_POOL_SIZE = 64;
-    public static final boolean DEFAULT_CACHE_COMPRESSION_METADATA = true;
+    @Deprecated
+    public static final boolean DEFAULT_CACHE_COMPRESSION_METADATA = false; // use org.apache.cassandra.spark.reader.SSTable cache
+    @Deprecated
     public static final long DEFAULT_MAX_SIZE_CACHE_COMPRESSION_METADATA_BYTES = 8 * MEBI_BYTES; // 8MiB
```

Comment:
Let's simply remove the fields that are newly introduced in the recent patch of CASSANDRA-19900